src/gluonts/dataset/dataframe.py

```diff
@@ -170,24 +191,25 @@ def as_dataentry(
 
     Parameters
     ----------
-    data: pandas.DataFrame
+    data:
         pandas.DataFrame containing at least ``timestamp``, ``target`` and
         ``item_id`` columns.
-    target: str or List[str]
+    target:
         Name of the column that contains the ``target`` time series.
         For multivariate targets ``target`` is expecting a list of column
         names.
-    timestamp: str
+    timestamp:
```

Comment:
Did you remove the type-annotations intentionally?

And wouldn't you then also need to remove the `:` as well?

Reply:
Thanks for spotting. Yes, I removed them intentionally. They already have an annotation with the type-hints. I removed them from the docstring to avoid mismatches between two different annotations and to follow the rest of gluonts code basis.